From: Dominique Leuenberger <dimstar@opensuse.org>

USe unsigned char in order to ensure we can store values up to (int)255

Fixes build with gcc 6, which failed with:
[  300s] Accents.cc:78:42: error: narrowing conversion of '128' from 'int' to 'char' inside { } [-Wnarrowing]
[  300s]     'o', 'u', 'u', 'u', 'u', 'y', 254, 255};


Index: htdig-3.2.0b6/htfuzzy/Accents.cc
===================================================================
--- htdig-3.2.0b6.orig/htfuzzy/Accents.cc
+++ htdig-3.2.0b6/htfuzzy/Accents.cc
@@ -43,7 +43,7 @@ extern int debug;
 | table iso-latin1 "minusculisee" et "de-accentuee" |
 `--------------------------------------------------*/
   
-static char MinusculeISOLAT1[256] = {
+static unsigned char MinusculeISOLAT1[256] = {
      0,   1,   2,   3,   4,   5,   6,   7,
      8,   9,  10,  11,  12,  13,  14,  15,
     16,  17,  18,  19,  20,  21,  22,  23,
